### ISSUE TITLE
Launcher fails to launch

### ISSUE THREAD

**Issue by DocterDum** [gorilla-devs/GDLauncher#930]
**Describe the bug**
Last night:
Double click shortcut -> Launcher opens
Today:
Double click shortcut -> Nothing happens

**To Reproduce**
Unsure

**Expected behavior**
Launcher opens

**Possible solutions**
Run C:\Users\<username>\AppData\Local\gdlauncher-updater\installer.exe -> After running this, the issue was resolved, even when running the normal exe

**Operating System:**
 - Windows 10 21.387.1 (Insider build) -> Updates have not run in the relevant period

**Additional context**
GDLauncher logs provided no helpful info

Steps I tried:
 - Launching via exe instead of shortcut
 - Restart PC
 - Run as admin
 - Close all other running apps

**Comment by ScottKillen** [comment]
I had an uncaught node.js exception that happened after the second and following launches this morning. I did not take a screenshot before solving the issue, but I remember the message something about "https:/github.com/" not being a valid URL.

Tried the above solution (running installer.exe) and GDLauncher launched correctly.

Upon launching the normal way, GDLauncher had forgotten all of my Mojang credentials (all my accounts were gone), and all of my custom settings (memory, java, etc.). The new user welcome screens displayed after I logged in; however, all of my instances were intact and I was able to re-add all of my Mojang accounts.

Running version 1.1.8
